clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -386,6 +386,14 @@ def CIR_PtrDiffOp : CIR_Op<"ptr_diff", [Pure, SameTypeOperands]> {
 //===----------------------------------------------------------------------===//
 // PtrStrideOp
 //===----------------------------------------------------------------------===//
+def CIR_PtrStrideAttr
+    : I32BitEnumAttr<"PtrStrideAttr", "Stride attribute for pointer arithmetic",
+                     [I32BitEnumAttrCaseNone<"None">,
+                      I32BitEnumAttrCase<"InBounds", 0x1, "inbounds">,
+                      I32BitEnumAttrCase<"NoUnsignedWrap", 0x2, "nuw">,
+                      I32BitEnumAttrCase<"NoSignedWrap", 0x4, "nsw">]> {
```

Comment:
Are those excluse or they can be set independently? If independent it might make more sense to do something like  `CIR_BinOp`